plugins/woocommerce/tests/api-core-tests/tests/shipping/shipping-zones.test.js

```diff
@@ -9,7 +9,7 @@ const shouldSkip = API_BASE_URL != undefined;
  * @group shipping
  */
 
-test.describe( 'Shipping zones API tests', () => {
+test.describe.serial( 'Shipping zones API tests', () => {
```

Comment:
This test suite is still a little flaky on occasion.  It looks like it's because these tests don't always run in order, and then some assertions in earlier tests fail due to data changes from later tests.  Something to keep an eye on.